wd.html

```diff
@@ -371,6 +371,79 @@ <h3>Internet Explorer</h3>
 <pre><code class=rb>require "selenium-webdriver"
 driver = Selenium::WebDriver.for :internet_explorer</code></pre>
 
+<h2>Browser Navigation</h2>
+
+<h3>Navigate To</h3>
+
+<p>The first thing you will want to do after launching a browser is to
+ open your website. This can be achieved in a single line:
+
+<pre>
+ <code class=java>
+//Convenient
+driver.get("https://www.seleniumhq.org");
+
+//Longer way
+driver.navigate().to("https://seleniumhq.github.io/docs/");
+ </code>
+ <code class=python>driver.get("https://www.seleniumhq.org")</code>
+ <code class=ruby>
+#Convenient
+driver.get "https://www.seleniumhq.org"
+
+#Longer way
+driver.navigate.to "https://seleniumhq.github.io/docs/"
```

Comment:
RSG way of writing this would be `driver.navigate.to('https://seleniumhq.github.io/docs/')` Same for the other method. Other than that the same is swish